src/sage/graphs/generators/families.py

```diff
@@ -2223,19 +2221,18 @@ def LCFGraph(n, shift_list, repeats):
     r"""
     Return the cubic graph specified in LCF notation.
 
-    LCF (Lederberg-Coxeter-Fruchte) notation is a concise way of
-    describing cubic Hamiltonian graphs. The way a graph is constructed
-    is as follows. Since there is a Hamiltonian cycle, we first create
-    a cycle on n nodes. The variable shift_list = [s_0, s_1, ...,
-    s_k-1] describes edges to be created by the following scheme: for
-    each i, connect vertex i to vertex (i + s_i). Then, repeats
-    specifies the number of times to repeat this process, where on the
-    jth repeat we connect vertex (i + j\*len(shift_list)) to vertex (
-    i + j\*len(shift_list) + s_i).
+    LCF (Lederberg-Coxeter-Fruchte) notation is a concise way of describing
+    cubic Hamiltonian graphs. The way a graph is constructed is as
+    follows. Since there is a Hamiltonian cycle, we first create a cycle on `n`
+    nodes. The variable `\textrm{shift_list} = [s_0, s_1, ..., s_k-1]`
+    describes edges to be created by the following scheme: for each `i`, connect
+    vertex `i` to vertex `(i + s_i)`. Then, repeats specifies the number of
+    times to repeat this process, where on the `j`-th repeat we connect vertex
+    `(i + j |\textrm{shift_list}|)` to vertex
+    `( i + j |\textrm{shift_list}| + s_i)`.
 
```

Comment:
As I understand this, `shift_list` and `repeats` should be code. Thus
```
The variable ``shift_list`` = `[s_0, s_1, ..., s_{k-1}]`
```
`s_k-1` seems an error. Hence the length of `shift_list` is `k`. Hence
```
Then, ``repeats`` specifies the number of
```
and
```
we connect vertex `i + jk` to vertex `i +jk + s_i`.
```
where i runs through 0, 1, ..., k - 1.

Reply:
I tried to improve it.